backend/common/utils/utils.py

```diff
@@ -78,9 +78,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def default(self, obj):
-        if isinstance(obj, np.float32):
+        if isinstance(obj, np.floating):
             return float(obj)
-        elif isinstance(obj, np.integer):
+        if isinstance(obj, np.integer):
```

Comment:
Curious, how does it handle the conversion of the `ndarray`? Or are we just using native python lists?

Reply:
via slicing.  ie,   

> type(np.zeros(10, dtype=np.float32)[0]) == np.float32

we aren't using native lists, but ndarray